workload-team/cluster-stamp.bicep

```diff
@@ -39,6 +39,9 @@ param gitOpsBootstrappingRepoHttpsUrl string = 'https://github.com/mspnp/aks-bas
 @minLength(1)
 param gitOpsBootstrappingRepoBranch string = 'main'
 
+@description('The AKS cluster Internal Load Balancer IP Address')
+param clusterInternalLoadBalancerIpAddress string = '10.240.4.4'
```

Comment:
It looks like this parameter isn't set in the calling script - should it be?

Reply:
I'm not quite sure whether it should @johndowns  but this can definitely be an alternative

Please let's analyze from the following commit what is the preference. Happy to revert or keep this as is. 

done | addressed from d5910b1145d0775319f2bed5491ba6b45d6c5797